### PR TITLE
Tree_Preorder_Traversal.js Added

### LINKED ISSUE #2550: Tree Preorder Traversal
- [x] C
- [x] C++
- [ ] C# - @sharmakeshav10
- [ ] CoffeeScript
- [x] Dart - @Rukmini-Meda
- [x] Go
- [x] Java
- [ ] JS - @afroz23
- [x] Kotlin - @PrachieNaik
- [x] Php
- [x] Python
- [ ] Readme - @SimranMakhija7
- [x] Ruby
- [ ] Rust
- [ ] TypeScript - @Shr03mink

### PR DESCRIPTION
**Pull request by afroz23**
Fixes #2550 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Tree_Preorder_Traversal.js Added 

#### Languages Used:
- JavaScript

#### Files Added:
- Tree_Preorder_Traversal.js
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by afroz23** [comment]
@singh-shreya6 mam review it :)

**Comment by singh-shreya6** [comment]
Squash your commits @afroz23. The code looks good.

File: Tree_Preorder_Traversal/Tree_Preorder_Traversal.js.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+/*
+output:
+Preorder Traversal:
+3
+1
+5
+4
+6
+
+*/
```
Comment (singh-shreya6) [inline Tree_Preorder_Traversal/Tree_Preorder_Traversal.js.js:98]: Please add a blank line at the end of file.

Comment (afroz23) [reply Tree_Preorder_Traversal/Tree_Preorder_Traversal.js.js]: Mam I am trying to squash all the commit but I am feeling very difficult, please help me out.

**Comment by singh-shreya6** [comment]
You are not squashing your commits, as you can see the number of commits have increased to 54, rather than decreasing to 1. If you are having trouble squashing, reach out to someone on slack or open a new PR

**Pull request closed without merging**